packages/lesswrong/lib/collectionUtils.js

```diff
@@ -36,7 +36,7 @@ async function conflictingIndexExists(collection, index, options)
 
 export async function ensureIndex(collection, index, options={})
 {
-  if (Meteor.isServer) {
+  if (Meteor.isServer && options) {
```

Comment:
This has a default value of `{}` so `options` should not, in practice, end up being null. But if `options` were in fact null, I think it would be better to crash, rather than silently skip creating the index.